src/metabase/api/card.clj

```diff
@@ -697,7 +697,8 @@ saved later when it is ready."
             (api/reconcile-position-for-collection! collection_id collection_position nil)
             ;; Now we can update the card with the new collection and a new calculated position
             ;; that appended to the end
-            (t2/update! Card (u/the-id card)
+            (t2/update! (t2/table-name Card)
```

Comment:
This isn't related to updating `db/update-where!`. Can you explain?

Reply:
I'm not 100% sure but without it moving cards will get a duplicated card_id error
`ERROR: duplicate key value violates unique constraint \"report_card_pkey\"\n  Detail: Key (id)=(590) already exists

try running this test `test-that-we-can-bulk-remove-some-cards-from-a-collection`

Reply:
I don't exactly remember why I change it. sorry.

but I'll revert it.